Makefile

```diff
@@ -3,7 +3,11 @@ install:
 	Rscript -e "renv::restore()"
 	Rscript -e 'renv::install(".")'
 
-.PHONY: all run run-nohup sync download upload dashboard
+test:
+	Rscript -e "testthat::test_dir('tests/testthat')"
+
+test-forecasters:
```

Comment:
I think as you've written it, it's going to skip the tests there, right? Or does not using `test_*` functions cause it to continue executing afterwards

Reply:
ah yea, this is outdated. to run test-forecasters-data.R, just uncomment the skip and run tests as normal. that way test-forecasters-data.R lives with the rest of the tests as normal and there's a nice message that lets the user know that there's a larger test available.

Reply:
right, skip does announce it's existence, so I suppose this works